pkg/admission/prep_downscale.go

```diff
@@ -185,6 +198,12 @@ func deny(msg string, args ...any) *v1.AdmissionResponse {
 }
 
 func getResourceAnnotations(ctx context.Context, ar v1.AdmissionReview, api kubernetes.Interface) (map[string]string, error) {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "Get resource annotations")
+	defer span.Finish()
+
+	span.SetTag("namespace", ar.Request.Namespace)
```

Comment:
We use `object.namespace` above. Should these match?

Reply:
Good catch, fixed in 1c1bffa